src/components/AudioPlayer/Player.js

```diff
@@ -623,16 +626,18 @@ class AudioPlayer extends Component {
             )}
             {!loading && title && fixed && (
               <div {...styles.time}>
-                <A style={{ color: colors.text }} href={sourcePath}>
+                <Link style={{ color: colors.text }} href={sourcePath} passHref>
                   {title}
```

Comment:
You still need the A tag:
```suggestion
                <Link href={sourcePath} passHref>
                  <A style={{ color: colors.text }} href={sourcePath}>
                    {title}
                  </A>
```

See [Next.js docs](https://nextjs.org/docs/api-reference/next/link) for a bit of background. Although it's not that well explained. I think it's worth reading the render method of the next.js Link component:
https://github.com/vercel/next.js/blob/e4f0848c81df060fb557271d3035bbd6d1ba13ed/packages/next/client/link.tsx#L238-L307 (ignore the url / href formatting parts, we used [next-routes](https://www.npmjs.com/package/next-routes) and custom logic which preprocesses this)

Reply:
Ah! I was just at the cusp of going crazy XD